news/125.bugfix

```diff
@@ -0,0 +1 @@
+[yurj] fix for https://github.com/plone/plone.app.users/issues/125 (view @@member-fields is public)
```

Comment:
Editing to use our normal convention. towncrier will add a link to the PR based on the number in the filename.

```suggestion
Check plone.app.controlpanel.UsersAndGroups permission for the @@member-fields edit view. @yurj 
```